oak/server/wasm_node.cc

```diff
@@ -561,4 +600,14 @@ wabt::interp::HostFunc::Callback WasmNode::OakRandomGet(wabt::interp::Environmen
   };
 }
 
+wabt::interp::HostFunc::Callback WasmNode::WasiPlaceholder() {
+  return [this](const wabt::interp::HostFunc* func, const wabt::interp::FuncSignature*,
+                const wabt::interp::TypedValues& args, wabt::interp::TypedValues& results) {
+    LogHostFunctionCall(name_, func, args);
+    LOG(QFATAL) << "{" << name_ << "} WASI is not implemented";
+    results[0].set_i32(OakStatus::ERR_INTERNAL);
+    return wabt::interp::Result::Ok;
```

Comment:
Also return an error here? Though I guess if it logged with `fatal` it would not actually reach here anyways.

Reply:
Done